fullstack_template/static/js/Login.jsx

```diff
@@ -43,27 +43,28 @@ class Login extends Component {
 		fetch('/login', {
 			method: 'POST',
 			dataType: 'json',
-			headers: {'Content-Type':'application/json'},
+			headers: { 'Content-Type': 'application/json' },
 			body: JSON.stringify(this.state)
-		}).then(function(response) {
-                	return response.json();
-                        //var promise = response.text();
-                        //console.log(promise);
-                        //console.log(promise.value);
+		}).then(function (response) {
+			if(response.status != 200) {
+				throw Error("Invalid Credentials")
+			}
+			return response.json();
+			//var promise = response.text();
+			//console.log(promise);
+			//console.log(promise.value);
 			//console.log("Current definition of app is:");
 			//console.log(app);
-                        
-		}).then(function(data) {
-                	const response = data;
-                        const json = JSON.parse(response);
-                	console.log(json);
-                        if(json['loginStatus'] === "successful login") {
-				app.handleLogin(self.state.username)
-                        }
-                        else {
-				app.handleLogin("")
-                        }
-                })
+		}).then(function (data) {
+			const json = data;
+			console.log(json)
+			//storing JWT to a cookie
+			document.cookie = "access_token=" + json["access_token"]

```

Comment:
Just wondering but is this cookie accessible throughout the user's whole session?

Reply:
Yes, it should be accessible throughout the whole session.